.gitignore

```diff
@@ -1,7 +1,26 @@
+# Tech report
 Simplicity-TR.pdf
-Haskell/*.pdf
 *.eps
+# Haskell
+/Haskell/*.pdf
+dist-newstyle
+# Nix
+result
 result-doc
+# Coq
 .lia.cache
 *.vok
 *.vos
```

Comment:
While we are at in can you add under `Coq`

    /Coq/*CoqMakefile*
    *.crashcoqide
    alectryon-doc